lua/starfall/libs_sh/entities.lua

```diff
@@ -1139,6 +1139,36 @@ function ents_methods:getModel()
 	return getent(self):GetModel()
 end
 
+--- Returns the entity's model bounds
```

Comment:
This should explain what it means by bounds; i.e. how does it differ from ent:obbMins() obbMax()